atlassian_jwt_auth/verifier.py

```diff
@@ -11,7 +13,7 @@ class JWTAuthVerifier(object):
     def __init__(self, public_key_retriever, **kwargs):
         self.public_key_retriever = public_key_retriever
         self.algorithms = algorithms.get_permitted_algorithm_names()
-        self._seen_jti = set()
+        self._seen_jti = deque(maxlen=1000)
```

Comment:
I don't mind having a backing deque like object but the (average) time complexity is for searching for a jti is higher for `deque` than it is for a `set` (O(n) vs O(1)).